content/docs/migration_1.0/Python.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Python"
+weight : 6500
+menu:
+  docs:
+    parent: migration_1.0
+---
+
+## Highlights
+
+The library has been fully rewritten to use only Rust. It should make no difference for users, except you may observe a significant performance improvement, up to x5.
+
+The API has also been reworked to feel more pythonic, using notably context managers.
+
+## Context managers and background execution
```

Comment:
Abrupt transition. Change from `RAII` to context manager should be briefly explained and only then describe the specific cases.

Reply:
@Mallets Can you add Joseph to this atolab ?

Reply:
Invitation sent to @wyfo.

Reply:
What do you call "abrupt transition"?

Why do we need to explain the change? The only thing users should modify in their code is adding a context manager for session, and that's clearly written on the first line; the rest of their code *can* remain the same. 

But I agree I should add a paragraph after the example of subscriber without context-manager to mention that keeping a variable in the scope is no more necessary.

Reply:
I think it's always good to explain the underlying change even in the case it does not require any change in the code of the user.

Reply:
I quite disagree here, especially here when we are talking about Python users who may not even know the acronym RAII or what is a Python finalizer. 
Also, there is no underlying change here, Python finalizers are still used, and you program may still deadlock if exit it on an exception. The only change was to implement context manager, to let users do things properly, and avoid deadlock issue.